InfinityPlay/Views/Shared/_Navigation.cshtml

```diff
@@ -0,0 +1,62 @@
+﻿<nav class="root-navbar">
+    <div class="navBar">
+
+        <div class="root-navbar-header">
+            <a class="navbar-brand" href="/home/index"><span class="logo">Infinity<span style="color:#fff;">Play</span></span></a>
+            @*@Html.ActionLink("InfinityPlay", "Index", "Home", new { area = "" }, new { @class = "navbar-brand" })*@
+        </div>
+        <ul id="navBar-Group-Links">
+            <li class="navBar-Group Home">
```

Comment:
Lot of nested nodes, might want to refactor

Reply:
Will refactor in Navigation Update branch